index.html

```diff
@@ -0,0 +1,316 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <link rel="stylesheet" href="./main.css">
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>NYTarticle</title>
+</head>
+
+<body>
+    <header class="flex">
+        <div id="left-header" class="inline-flex">
+            <button><img src="./hamburger-menu-lines-icon.png" alt="burger-menu"></button>
+            <button id="search-button"><img src="./index.png" alt="search-button"></button>
+            <p id="hidden-p">SPACE & COSMOS</p>
+        </div>
+
+        <div id="mid-header">
+            <img src="./ny-times-logo.png" alt="main-header-logo">
+        </div>
+
+        <div id="right-header" class="inline-flex">
+            <button>SUBSCRIBE NOW</button>
+            <button>LOG IN</button>
+        </div>
+    </header>
+    <main>
+        <article>
+            <section id="section-1">
+                <div id="second-div">
+                    <div>
+                        <h1 id="title"><i>Space Ripples Reveal Big Bang's Smoking Gun</i></h1>
+                    </div>
+                    <div id="main-image">
+                        <figure>
+                            <img src="https://static01.nyt.com/images/2014/03/18/science/space/18cosmos/18cosmos-superJumbo.jpg?quality=90&auto=webp" alt="Alan Guth">
+                            <figcaption>Alan Guth was one of the first physicists to hypothesize the existence of inflation, which explains how the universe expanded
+                                <br> so uniformly and so quickly in the instant after the Big Bang 13.8 billion years ago.Credit.<span class="credit">Rick Friedman for The New York</span> Times</figcaption>
+                        </figure>
+                    </div>
+                </div>
+            </section>
+            <section id="section-2">
+                <div>
+                    <div class="new-line"></div>
+
```

Comment:
### ADD missing sections in the main article.
- [ ] There are some missing social links at the top of the main article's image. Do you mind adding them?

|[Source Page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html)|[Clone Page](https://raw.githack.com/igorkol91/NYTarticle/newbranch/index.html)|
|:--:|:--:|
|![Main-Article-socials-source](https://user-images.githubusercontent.com/22345961/112321452-11ca9d00-8cc1-11eb-909a-1ba1beff5725.png)|![Main-Article-socials](https://user-images.githubusercontent.com/22345961/112321467-1727e780-8cc1-11eb-929d-55b76f99bd54.png)|